platforms/arm.go

```diff
@@ -0,0 +1,69 @@
+package platforms
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"runtime"
+	"strings"
+)
+
+// Structure to identify ARM platforms
+type PlatformArm struct {
+	// Present the ARM instruction set architecture, eg: v7, v8
+	Variant string
+	// Present the features that could not be detectable through HWCAPS
+	Features []string
+}
+
+func isArmArch(arch string) bool {
```

Comment:
Functions like this belong in `database.go`.

Reply:
Ok, I will move it to database.go